lib/SPIRV/libSPIRV/SPIRVInstruction.h

```diff
@@ -2003,6 +2003,12 @@ class SPIRVExtInst : public SPIRVFunctionCallGeneric<OpExtInst, 5> {
     }
     return VArgs;
   }
+  SPIRVValue *getArg(SPIRVWord I) {
```

Comment:
I think this should be renamed getArgValue to be consistent with the pre-existing getArgValues which returns a vector pointers.  Also, consider implementing getArgValues with getArgValue.